### PR TITLE
Desktop version (Responsive Design)

### PR DESCRIPTION
**Pull request by williamscch**
In this project, I used media queries in the CSS file to make my Desktop version look like the template I selected from Figma (3rd one) 
The media queries start to work with the min-width: 768px or wider.

In order to lay out the elements on the page, I used Grid and Flexbox. I did my best to replicate as much as possible with the original design --> [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -212,18 +219,21 @@ <h3>Skills</h3>
         </ul>
     </section>
     <footer>
```
Comment (tufoinnkuo10) [inline index.html:222]: - [x] please kindly reduce the space between your skill section cards 
your design 
Screenshot (97)_LI
figma design
Screenshot (99)

File: index.html [hunk 2]
```diff
@@ -212,18 +219,21 @@ <h3>Skills</h3>
...
+                <input type="email" name="email" required="required" placeholder="Email address">
 
-            <input type="text" name="full-name" required="required" maxlength="30" placeholder="Full Name">
-    
-            <input type="email" name="email" required="required" placeholder="Email address">
-    
-            <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
-    
-            <button type="submit">Get in touch</button>
-        </form>
```
Comment (tufoinnkuo10) [inline index.html:225]: - [x] kindly add a little bit of padding to your desktop form placeholders so the get the position of the original figma version

File: index.html [hunk 3]
```diff
@@ -212,18 +219,21 @@ <h3>Skills</h3>
...
+                <textarea name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+
+                <button type="submit">Get in touch</button>
+            </form>
+        </div>
         <div class="network">
+            <hr/>
             <ul>
```
Comment (tufoinnkuo10) [inline index.html:238]: - [x] please add a medium social icon. it will be of help when you start writing your articles

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @wdavidcch 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ linter error checks validated
✅ Documented your work professionally

Required Changes ♻️
Check the comments under the review
almost
Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: assets/css/style.css [hunk 4]
```diff
@@ -22,18 +23,36 @@ body {
...
 
-.mobile ul {
+.desktop {
   display: flex;
-  justify-content: space-between;
+  justify-content: center;
+  gap: 40px;
+  margin-top: 0;
+  padding-top: 16px;
+  padding-left: 0;
```
Comment (Kandy-Peter) [inline assets/css/style.css:47]: - [x] OPTIONAL: I would suggest you make your cards more responsive and improve your design quality. You can take a look at the code below:
```
.desktop ul {
   display:grid;
   grid-template-column: repeat(auto-fit, minmax(350px, 1fr);
}
```

**Review by Kandy-Peter — APPROVED** [review]
Hi @wdavidcch,

# APPROVED 🟢 

Great implementation of all the requirements in this milestone. 👏

Your web app looks really fantastic. 🎉 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

#### CONGRATULATIONS! 🎉 

##  Optional suggestion

- [x] Please, ensure that you include an explanation, a "why", for each requested change.

# CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 4 above)
Comment (williamscch) [reply assets/css/style.css:47]: Okay thank you

**Pull request merged**